maintainers/working_groups/learning_journey/tutorial-outlines/first-steps-trivial-builders.md

```diff
@@ -0,0 +1,37 @@
+# First Steps - Trivial Builders
+
+## Assumptions
+
+* The reader knows what a shell environment is.
+* The reader has some familiarity with `nix-shell`
+
+## Activity
+
+* Start with a `shell.nix` which brings in some packages and dependencies for a `Process X`
+  * `Process X` should be capable of accepting a config file
+* Show the reader how to create a series of utility scripts for starting/stopping and generally streamlining everyday dev tasks with `Process X` and demonstrate how the config file can be generated with Nix.
+
+## Purpose
+
+* Show the reader how trivial builders are the everyday glue that makes writing devshells, packages and NixOS modules easier, cleaner and simpler to follow. 
+
+## Steps
+
+* Begin with a `shell.nix` which provides the dependencies for `Process X`
+* Show the reader how to generate the config file using `writeTextFile`
+* Show the reader how to take the generated config file and create a start script for `Process X` using `writeScript`
+  * When they reload their shell and try to run the new script, they should see that it fails
+  * This is a good opportunity to introduce `writeScriptBin` and highlight how it is necessary to ensure their script is placed in `$out/bin` to be picked up by `mkShell`.
+* Having created a working start script using `writeScriptBin` we point out that the generated script does not contain an interpreter:
```

Comment:
It's not clear what is meant by missing interpreter. Is this a runtime dependency or something else?